doc/schemas/options_schema.json

```diff
@@ -301,6 +313,7 @@
     "Terrains",
     "Cliques",
     "AlphaCalculatorMode",
-    "CombinerModuleGroup"
+    "CombinerModuleGroup",
+    "DebugTileResolution"
```

Comment:
Done. User can specify an exact resolution, otherwise it defaults to `tile size * (8,8)`.

Perhaps the user should just specify a multiplier; in that case it should probably be a separate issue.